Add reference to Pairing new bulbs (#368)
sidoh/esp8266_milight_hub: README.md

![Web UI](https://user-images.githubusercontent.com/589893/39412360-0d95ab2e-4bd0-11e8-915c-7fef7ee38761.png)

#### Pair Bulbs

> Of course you need to pair some bulbs, how to do this is [described in the wiki](https://github.com/sidoh/esp8266_milight_hub/wiki/Pairing-new-bulbs).

Reviewer: ```suggestion
If you need to pair some bulbs, how to do this is [described in the wiki](https://github.com/sidoh/esp8266_milight_hub/wiki/Pairing-new-bulbs).
```

Reviewer: Adding the hedge since I think people replacing existing hardware won't need to pair bulbs.
